root/defaults/autostart

```diff
@@ -1,22 +1,2 @@
 #! /bin/bash
-
-# Only start in no-sandbox mode on unpriv systems
-if grep -q 'Seccomp:	0' /proc/1/status; then
-  chromium-browser \
-    --ignore-gpu-blocklist \
-    --no-first-run \
-    --password-store=basic \
-    --simulate-outdated-no-au='Tue, 31 Dec 2099 23:59:59 GMT' \
-    --start-maximized \
-    --user-data-dir > /dev/null 2>&1
-else
-  chromium-browser \
-    --ignore-gpu-blocklist \
-    --no-first-run \
-    --no-sandbox \
-    --password-store=basic \
-    --simulate-outdated-no-au='Tue, 31 Dec 2099 23:59:59 GMT' \
-    --start-maximized \
-    --test-type \
-    --user-data-dir > /dev/null 2>&1
-fi
+chromium-browser ${CHROME_CLI}
```

Comment:
for the sake of being extra careful with bash, this should technically be something more like:
```
chromium-browser "${CHROME_CLI[@]}"
```
but that requires some additional logic to split `CHROME_CLI` from a space separated string into an array in bash. A quick mock would look something like this:
```
CHROME_CLI_ARGS=()
while IFS= read -r -d '' line; do
    CHROME_CLI_ARGS+=("${line}")
done < <(echo "${CHROME_CLI}" | tr " " "\n")
chromium-browser "${CHROME_CLI_ARGS[@]}"
```
but this comes with a potential disadvantage where arguments and their values cannot contain spaces (it would cause improper splitting, which may or may not be an issue), ex:
```
--arg1 valA --arg2 "val B"
```
In theory, how you have it should work, but only because bash is lax as hell, and there's no guarantee it won't behave strangely in specific scenarios.

Reply:
I actually tested this pretty well and specifically left the quotes off of it. Makes it work more like eval, it literally passes the commands the user puts in the string. I think any of our efforts to sanitize this input would just break things or be a burden to the user requiring escape characters.